### DISCUSSION TITLE
sensitivity score

### DISCUSSION THREAD

**Discussion by sophieleech** [FrederickHuangLin/ANCOMBC#160] [Q&A]
Hi, 

I am not sure how to interpret the sensitivity scores given in ANCOM-BC2. 

In the tutorial https://bioconductor.org/packages/release/bioc/vignettes/ANCOMBC/inst/doc/ANCOMBC2.html it says that extreme sensitivity score were defined as scores >4. How was this number decided on and is it specific to the tutorial data? Or can this be applied generally? 

Cheers!

**Comment by FrederickHuangLin** [comment]
Hi @sophieleech,

Based on our simulation benchmark, setting the cutoff value at one will almost always guarantee the FDR below the nominal level while setting the cutoff value at four will make the test more powerful but sometimes lose control of the FDR. Unfortunately, this is more like an ad-hoc way of the FDR control, we do not have theoretical justifications for the cutoff value.

Best,
Huang
